src/util/chia_logging.py

```diff
@@ -10,6 +10,8 @@
 
 def initialize_logging(service_name: str, logging_config: Dict, root_path: Path):
     log_path = path_from_root(root_path, logging_config.get("log_filename", "log/debug.log"))
+    log_date_format = "%Y-%m-%d %H:%M:%S"
```

Comment:
May I suggest having a T `%Y-%m-%dT%H:%M%S` for ISO 8601 compliance?

Reply:
:+1: